pkg/iac-providers/terraform/v12/resource.go

```diff
@@ -19,12 +19,18 @@ package tfv12
 import (
 	"fmt"
 	"io/ioutil"
+	"regexp"
+	"strings"
 
+	"github.com/accurics/terrascan/pkg/iac-providers/output"
 	"github.com/hashicorp/hcl/v2/hclsyntax"
 	hclConfigs "github.com/hashicorp/terraform/configs"
 	"go.uber.org/zap"
+)
 
-	"github.com/accurics/terrascan/pkg/iac-providers/output"
+var (
+	skipRulesPattern = regexp.MustCompile(`#ts:skip=\s*(([A-Za-z0-9]+\.?){5})(\s*,\s*([A-Za-z0-9]+\.?){5})*`)
```

Comment:
Just FYI, I think the rule ID format may go through some changes prior to the next release.

Reply:
Yes, kept that in mind!